src/Formidable.ml

```diff
@@ -78,7 +78,7 @@ module Props = struct
       setStatus : 'error States.Field.Status.t -> unit;
       status : 'error States.Field.Status.t;
       validate : unit -> unit;
-      validationNames : string list;
+      hasValidation :  string -> bool;
```

Comment:
Weird a space just appeared, we still have issues with the formatter :(
(If you can't solve it, it's ok 👍 )

Reply:
thank you I fixed it, yeah sometime my formatter is not working actually